### ISSUE TITLE
[Help]: Want to ask about TTA task inference, why we need to add uncond_embeddings

### ISSUE THREAD

**Issue by hieuhthh** [open-mmlab/Amphion#215]
`    def get_text_embedding(self):
        text = self.args.text

        prompt = [text]

        text_input = self.tokenizer(
            prompt,
            max_length=self.tokenizer.model_max_length,
            truncation=True,
            padding="do_not_pad",
            return_tensors="pt",
        )
        text_embeddings = self.text_encoder(
            text_input.input_ids.to(self.args.local_rank)
        )[0]

        max_length = text_input.input_ids.shape[-1]
        uncond_input = self.tokenizer(
            [""] * 1, padding="max_length", max_length=max_length, return_tensors="pt"
        )
        uncond_embeddings = self.text_encoder(
            uncond_input.input_ids.to(self.args.local_rank)
        )[0]
        text_embeddings = torch.cat([uncond_embeddings, text_embeddings])

        return text_embeddings`
        
        
        In file audioldm_inference.py, when we get text embedding, why we need to concat with uncond, what does it mean and do we really need it. Thank you!
